src/rules/at-rule-empty-line-before/__tests__/index.js

```diff
@@ -121,6 +121,35 @@ testRule(rule, mergeTestDescriptions(sharedAlwaysTests, {
 
 testRule(rule, {
   ruleName,
+  config: [ "always", { ignore: ["blockless-group"] } ],
+
+  accept: [{
+    code: "@media {}; @import 'x.css';",
+  }],
+
+  reject: [{
+    code: "@import 'x.css'; @media {};",
+    message: messages.expected,
+  }],
+})
+
+testRule(rule, {
```

Comment:
No need for these SCSS tests, as they don't contain any syntax that would violate the CSS parser, right? Unless I'm wrong, I'd suggest just adding the `@content;` line to a test in the block above and eliminating the SCSS tests.